rentoshare-jdl.jdl

```diff
@@ -0,0 +1,137 @@
+
+application {
+  config {
+    baseName gateway // (1)
+    reactive true // (2)
+    packageName com.michael.developer.gateway
+    applicationType gateway
+    authenticationType oauth2 // (3)
+    buildTool maven // (4)
+    clientFramework angular // (5)
+    prodDatabaseType postgresql
+    serviceDiscoveryType eureka
+    testFrameworks [cypress] // (6)
+  }
+  entities Blog, Post, Tag, Property
+}
+
+application {
+  config {
+    baseName blog
+    reactive true
+    packageName com.michael.developer.blog
+    applicationType microservice // (7)
+    authenticationType oauth2
+    buildTool maven
+    databaseType mongodb
+    devDatabaseType mongodb
+    prodDatabaseType mongodb
+    enableHibernateCache false
+    serverPort 8081
+    serviceDiscoveryType eureka
+  }
+  entities Blog, Post, Tag
+}
+
+application {
+  config {
+    baseName store
+    reactive true
+    packageName com.michael.developer.store
+    applicationType microservice
+    authenticationType oauth2
+    buildTool maven
+    databaseType mongodb
+    devDatabaseType mongodb
+    prodDatabaseType mongodb
+    enableHibernateCache false
+    serverPort 8082
+    serviceDiscoveryType eureka
+  }
+  entities Property, Image
+}
+application {
```

Comment:
just for consistency, can you put an extra-line between l.52 and l.53 please?

Reply:
and between every `entity`, `relationship`, options, etc.

thanks

Reply:
Will do that.
Thanks